### PR TITLE
Easy compile script

### PR DESCRIPTION
**Pull request by SafwatHalaby**
Migrated Cubedown to this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,11 @@ For Windows, you just need to download a file and extract it:
  - [Windows 32 bit](http://builds.cuberite.org/job/MCServer%20Windows%20x86/lastSuccessfulBuild/artifact/Install/MCServer.zip)
  - [Windows 64 bit](http://builds.cuberite.org/job/MCServer%20Windows%20x64/lastSuccessfulBuild/artifact/Install/MCServer.zip)
 
-For other operating systems you need to download and compile yourself. There is also an archive of binary builds on the buildserver: http://builds.cuberite.org
+For other operating systems you need to download and compile yourself. This can be done either manually, or with this automatic script:
+
+    bash -c "$(wget -O - https://raw.githubusercontent.com/SafwatHalaby/cubeDown/master/cubeDown.sh)"
```
Comment (SafwatHalaby) [inline README.md]: Need to update url.

Comment (SafwatHalaby) [inline README.md]: Fixed, squashed.

**Comment by bearbin** [comment]
You should update the paragraph in the book to reflect the new URL.

**Comment by SafwatHalaby** [comment]
And in the forums.
Will do.

**Pull request merged**